webapp/client/src/components/FormFieldDropDown.test.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import FormFieldDropDown from "./FormFieldDropDown";
+
+describe("FormFieldDropDown", () => {
+  let props;
+  const onChangeHandler = jest.fn();
+
+  beforeEach(() => {
+    props = {
+      fieldName: "fooName",
+      fieldId: "fooId",
+      label: {
+        text: "foo",
+      },
+      errors: [],
+      options: [
+        { value: "A", displayName: "Vulcan" },
+        { value: "B", displayName: "Terra" },
+        { value: "C", displayName: "Earth" },
+      ],
+      onChangeHandler: onChangeHandler,
+    };
+    render(<FormFieldDropDown {...props} />);
+  });
+
+  describe("When second option selected", () => {
+    beforeEach(() => {
+      userEvent.selectOptions(screen.getByRole("combobox"), ["B"]);
+    });
```

Comment:
Do we need a test for changing the value?

Reply:
Great idea 👍

Reply:
This lead me to a rabbit hole because the tests for interacting with the Dropdown component via keyboard did not really work as expected. I was not able to get the tests running with the user event library. I have opened an issue in their repository and I will come back to this when ever there is something to follow up and add correct tests for the interaction via keyboard